frontend/lib/norent/components/social-icons.tsx

```diff
@@ -13,16 +15,18 @@ const socialMediaPageLinks = [
   { name: "instagram", url: "https://www.instagram.com/justfixnyc/" },
 ];
 
-const prefilledTwitterCopy =
-  "No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. " +
-  "This free tool sends a certified letter informing them of your protections. " +
-  "Join the #cancelrent movement at norent.org.";
-const prefilledEmailSubject =
-  "Just used JustFix.nyc's new free tool to tell my landlord I can't pay rent";
-const prefilledEmailBody =
-  "I used www.norent.org to tell my landlord that I'm unable to pay this month's rent. " +
-  "This free tool helps you build and send a letter to your landlord, cites legal protections in your state, " +
-  "and connects you to other people in your community working to #cancelrent";
+const prefilledTwitterCopy = () =>
+  li18n._(
+    t`No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. This free tool sends a certified letter informing them of your protections. Join the #cancelrent movement at norent.org.`
+  );
+const prefilledEmailSubject = () =>
+  li18n._(
+    t`Just used JustFix.nyc's new free tool to tell my landlord I can't pay rent`
+  );
+const prefilledEmailBody = () =>
+  li18n._(
+    t`I used www.norent.org to tell my landlord that I'm unable to pay this month's rent. This free tool helps you build and send a letter to your landlord, cites legal protections in your state, and connects you to other people in your community working to #cancelrent`
+  );
```

Comment:
It might be useful to use that `defineMessage` macro I mentioned earlier to explain what context these strings will be shown in?  The names of the variables here are helpful to me reading this, so I wonder if comments would serve a similar help for translators.  I dunno.